cryptol-remote-api/Dockerfile

```
@@ -110,5 +110,5 @@ USER cryptol
 ENV LANG=C.UTF-8 \
     LC_ALL=C.UTF-8
 ENTRYPOINT ["/usr/local/bin/cryptol-remote-api"]
-CMD ["http", "--host", "0.0.0.0", "--port", "8080", "/"]
+CMD ["http", "--host", "0.0.0.0", "--port", "8080", "/home/cryptol"]
```

That's the HTTP path, not the CWD. Do this instead:

```suggestion
WORKDIR /home/cryptol
CMD ["http", "--host", "0.0.0.0", "--port", "8080", "/"]
```

---

Thanks!